stdlib/public/stubs/OptionalBridgingHelper.mm

```diff
@@ -36,12 +36,29 @@ @interface __SwiftNull : NSObject {
 }
 @end
 
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_API
+id swift_stdlib_boxUTF8CString(const char *cstr, int len);
+
 @implementation __SwiftNull : NSObject
 
-- (NSString*)description {
-  return [NSString stringWithFormat:@"<%@ %p depth = %u>", [self class],
-                                                           (void*)self,
-                                                           self->depth];
+//   int
+ // asprintf(char **ret, const char *format, ...);
+
+- (id)description {
+  char *str = NULL;
+  const char *clsName = class_getName([self class]);
+  int fmtResult = asprintf(&str, "<%s %p depth = %u>", clsName,
+                                                       (void*)self,
+                                                       self->depth);
+  id result = nil;
+  if (fmtResult != -1) {
+    result = swift_stdlib_boxUTF8CString(str, strlen(str));
+    free(str);
+  } else {
+    const char *errMsg = "unable to format description of null";
```

Comment:
Possibly I should just abort if asprintf fails, tbh

Reply:
What's the "null" in this case?

Reply:
This is the "nil as AnyObject" code

Reply:
Oh. Yeah, that seems very unlikely to fail, since they're all classes we control.